### ISSUE TITLE
Warning about cloud-config while upgrading

### ISSUE THREAD

**Issue by robbertkl** [rancher/os#1610]
**RancherOS Version: (ros os version)**
Upgrade from v0.7.0 to v0.8.0

**Where are you running RancherOS? (docker-machine, AWS, GCE, baremetal, etc.)**
VPS

Got this message when upgrading from v0.7.0 to v0.8.0 today:

```
os-upgrade_1 | > time="2017-02-15T15:41:37Z" level=warning msg="Cloud-config not provided: you might need to provide cloud-config on bootDir with ssh_authorized_keys" 
```

I do have a cloud-config somewhere from my initial install. I ignored the warning and everything was fine after the upgrade/reboot: I could still SSH using my keys.

**Comment by SvenDowideit** [comment]
👍 for ignoring it - sorry, I rewrote the installer and upgrade code, and got so used to seeing that complaint that I totally didn't notice that it made no sense to have it for upgrade.

**Comment by SvenDowideit** [comment]
this will be fixed in v0.8.1
